lighting_import_attachment_smb/models/lighting_import_attachment_backend.py

```diff
@@ -0,0 +1,30 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class LightingImportAttachmentBackend(models.Model):
+    _name = "lighting.import.attachment.backend"
+    _description = "Lighting Import Attachment Backend"
+
+    name = fields.Char()
+    host = fields.Char()
+    user = fields.Char()
+    password = fields.Char()
+    port = fields.Integer()
+    folder = fields.Char()
+    default = fields.Boolean()
+    state = fields.Selection(
+        selection=[
+            ("draft", "Draft"),
+            ("validated", "Validated"),
+        ],
+        default="draft",
+    )
```

Comment:
Define `required`, `tracking`, `cascade`, etc... and sql constraints (if any) at the very beginning of coding, if you don't there's a very high chances you forget.